README.md

```diff
@@ -1 +1,11 @@
 # webui
+
+## Development
+
+To set up a development environment
+
+1. Install `go` v1.15
+2. Install Node.js version 14.15.1
+3. Install `gin`: `go get github.com/codegangsta/gin`
+4. `make dev` to compile and run the app
+5. `yarn start` to start the frontend dev server (with hot-reloading)
```

Comment:
Technically a requirement, as it is not installed by default like `npm`? Also, not really sure `yarn` still offers many advantages over [`npm` v7](https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/).

Additional note from the original author: https://twitter.com/sebmck/status/1300655408688709633
Additional alternative: https://pnpm.js.org/en/motivation

Reply:
Removed `yarn` as a dependency and updated the docs.